setup.cfg

```diff
@@ -4,6 +4,11 @@ version = attr: kaitaistruct.__version__
 author = Kaitai Project
 author_email = greycat@kaitai.io
 url = https://kaitai.io
+project_urls =
+    Source = https://github.com/kaitai-io/kaitai_struct_python_runtime
+    Tracker = https://github.com/kaitai-io/kaitai_struct_python_runtime/issues
```

Comment:
We can also add some "Documentation" link, but I'm not sure if it should link to https://doc.kaitai.io/, https://doc.kaitai.io/lang_python.html or somewhere else.